packages/wonder-blocks-data/src/util/__tests__/response-cache.test.js

```diff
@@ -112,779 +108,448 @@ describe("../response-cache.js", () => {
                 // Arrange
-                jest.spyOn(Server, "isServerSide").mockReturnValue(false);
-                const internalCache = new MemoryCache();
-                jest.spyOn(internalCache, "store");
-                const cache = new ResponseCache(internalCache);
-                const customCache = {
-                    store: jest.fn(),
-                    retrieve: jest.fn(),
-                    remove: jest.fn(),
-                    removeAll: jest.fn(),
-                };
+                const hydrationCache = new MemoryCache();
+                const ssrOnlyCache = new MemoryCache();
+                const cache = new ResponseCache(hydrationCache, ssrOnlyCache);
                 const fakeHandler: IRequestHandler<string, string> = {
                     getKey: () => "MY_KEY",
                     type: "MY_HANDLER",
-                    shouldRefreshCache: () => false,
                     fulfillRequest: jest.fn(),
-                    cache: customCache,
                     hydrate: true,
                 };
+                const ssrOnlyStoreSpy = jest.spyOn(ssrOnlyCache, "store");
 
                 // Act
                 cache.cacheData(fakeHandler, "options", "data");
 
                 // Assert
-                expect(customCache.store).toHaveBeenCalledWith(
-                    fakeHandler,
-                    "options",
-                    {data: "data"},
-                );
-                expect(internalCache.store).not.toHaveBeenCalled();
+                expect(ssrOnlyStoreSpy).not.toHaveBeenCalled();
             });
         });
 
-        describe("no hydrate", () => {
-            it("should store the entry in the ssr-only cache", () => {
-                // Arrange
-                jest.spyOn(Server, "isServerSide").mockReturnValue(true);
-                const ssrOnlyCache = new MemoryCache();
-                const internalCache = new MemoryCache();
-                const storeSpy = jest.spyOn(ssrOnlyCache, "store");
-                const cache = new ResponseCache(internalCache, ssrOnlyCache);
-                const fakeHandler: IRequestHandler<string, string> = {
-                    getKey: () => "MY_KEY",
-                    type: "MY_HANDLER",
-                    shouldRefreshCache: () => false,
-                    fulfillRequest: jest.fn(),
-                    cache: null,
-                    hydrate: false,
-                };
-
-                // Act
-                cache.cacheData(fakeHandler, "options", "data");
-
-                // Assert
-                expect(storeSpy).toHaveBeenCalledWith(fakeHandler, "options", {
-                    data: "data",
-                });
-            });
-
-            it("should not store the entry in the framework cache", () => {
-                // Arrange
+        describe("when server-side", () => {
+            beforeEach(() => {
                 jest.spyOn(Server, "isServerSide").mockReturnValue(true);
```

Comment:
This feels like a good use of shared `Arrange` since it's literally in the title of the describe block.